components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -113,29 +114,21 @@ export default function OrganizationSelector() {
             ),
             // marking as active for styles
             active: true,
-            separator: true,
+            separator: false,
```

Comment:
**suggestion**: We need this but shouldn't wrap with the rounded corners. Separator line should go from the left to right edge of the dropdown.